Day-6-7/src/components/UserCard.js

```diff
@@ -25,32 +22,29 @@ const GithubCard = ({ data }) => {
       });
   };
 
+  console.log(teamData);
+
+  if (teamData && teamData.length === 0)
+    return <div className="loader"> Loading...</div>;
+
   return (
     <div className="card-container">
       {teamData.map((item) => {
```

Comment:
after .map, in argument, have `{id, login...}`, then you can remove item.name, etc.